target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -121,6 +121,10 @@ pcs,cr3000)
 qihoo,c301)
 	ucidef_set_led_wlan "wlan" "WLAN" "$boardname:green:wlan" "phy0tpt"
 	;;
+telco_electronics,tel-t1)
```

Comment:
We already have a telco electronics device in ramips:
https://github.com/openwrt/openwrt/commit/73e0f52b6e4e1c79e4e95573f2050ae9ff10ac00

Based on that, and since it replaces a space, the vendor part of the device name should be _telco-electronics_.

When googling, I only find Telco Electronics T1. So, the whole string IMO should be _telco-electronics,t1_ ...